src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/components/api-authorization/AuthorizeRoute.js

```diff
@@ -1,4 +1,4 @@
-﻿import React from 'react'
+import React from 'react'
 import { Component } from 'react'
 import { Route, Redirect } from 'react-router-dom'
 import { ApplicationPaths, QueryParameterNames } from './ApiAuthorizationConstants'
```

Comment:
Nit: Missing semicolons at the ends of all these lines.

Reply:
I'm wondering why the linter doesn't complain